packages/carbon-graphs/src/Graphs.module.scss

```diff
@@ -1,9 +1,11 @@
+// eslint-disable terra/custom-property-namespace 
```

Comment:
Why are you moving these files from the terra-graphs package? React components should remain in the terra-graphs package.

Reply:
Sorry, things got mixed up in the initial confusion about the package names. I removed all react components from vanilla carbon here: https://github.com/cerner/terra-graphs/pull/5/commits/15f0fd97b18016ea3195adc535dac4f7ff503648